Documentation/building/unix-instructions.md

```diff
@@ -13,29 +13,53 @@ For more information about the different options when building, run `build.sh -?
 
 ## Minimum Hardware Requirements
 - 2GB RAM
+- x64
 
-## Prerequisites (native build)
+## Prerequisites
 
 ### Linux
 
-First, the package lists might need to be updated
+For Ubuntu, the following components are needed:
```

Comment:
We get pretty heavily into Ubuntu here, but no other distro.  It almost makes it sound like we only support building on Ubuntu, which isn't the case.

Can we just list our direct dependencies (git, cmake with minimum version, clang with minimum version), and then say `And all the requirements necessary to run .NET Core 2.0 applications` with the link to the docs site?

Reply:
I agree with you, my intention was to spell out the commands and to be consistent with corefx (there is also only instructions for ubuntu).

I'll push up another commit to address this feedback.